### PR TITLE
Migration Assistant docs page

### PR DESCRIPTION
**Pull request by bgrenon**
https://neon-next-git-bgrenon-migration-assistant-neondatabase.vercel.app/docs/import/migration-assistant

### CODE REVIEW

File: content/docs/import/migration-assistant.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+title: Neon Migration Assistant
+subtitle: Move your existing database into Neon using our guided tool
```
Comment (danieltprice) [inline content/docs/import/migration-assistant.md:3]:
```suggestion
subtitle: Move your existing database to Neon using our guided tool
```

File: content/docs/import/migration-assistant.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+enableTableOfContents: true
+---
+
+When you're ready to move your data into Neon, our Migration Assistant can help. Aimed at databases in the 100 to 200 GiB range, you need only the connection string to your existing database to get started.
```
Comment (danieltprice) [inline content/docs/import/migration-assistant.md:7]:
```suggestion
When you're ready to move your data to Neon, our Migration Assistant can help. Aimed at databases in the 100 to 200 GiB range, you need only the connection string to your existing database to get started.
```

Comment (danieltprice) [reply content/docs/import/migration-assistant.md:7]: Not sure if we should suggest a size. Could be used for any, really.

File: content/docs/import/migration-assistant.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  <img src="/docs/import/migration_string.png" alt="paste connection string for source db" style={{ width: '80%', maxWidth: '600px', height: 'auto' }} />
+</div>
+
+Neon will check the availability and configuration of your source database to help make sure your migration will be successful:
+
+- **Postgres version** — Verifies that your source database uses a version of Postgres that Neon supports (Postgres 14 to 17).
+- **Region** — Checks the hosting region of your source database.
+- **Supported extensions** — Identifies whether your extensions are supported by Neon. Unsupported extensions are listed, but you are not blocked from continuing the migration. Use your discretion.
+- **Compatible extensions** — Checks that your extension versions match Neon's current support. See [Supported Postgres extensions](/docs/extensions/pg-extensions) for a matrix of extensions to Postgres versions in Neon.
+- **Logical replication** — If logical replication is not enabled on your source database, you will see a warning message. You'll need to enable it if you intend to set up replication as part of the migration.
```
Comment (danieltprice) [inline content/docs/import/migration-assistant.md:51]:
```suggestion
- **Logical replication** — If logical replication is not enabled on your source database, you will see a warning message. You'll need to enable it if you intend to set up replication as part of the migration. Enabling logical replication in Postgres typically requires setting `wal_level=logical`. Check your provider's documentation for instructions.
```

File: content/docs/import/migration-assistant.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+<div style={{ display: 'flex', alignItems: 'top' }}>
+  <div style={{ flex: '0 0 55%', paddingRight: '20px' }}>
+    By default, we try to create your new project to match your source database:
+
+    - Matching Postgres **version**
+    - Matching **region**
+
+      <Admonition type="note">
+      This is an early feature and not work for all regions or providers.
```
Comment (danieltprice) [inline content/docs/import/migration-assistant.md:63]:
```suggestion
      This is an early feature and may not work for all regions or providers.
```

**Comment by danieltprice** [comment]
One thing that's not clear is whether we're presenting two methods for moving data (1. pg_dump & pgrestore, 2. logical replication) or if you're supposed to do the dump and restore and then follow that with LR to catch up or continue moving data. I think we should make that clear in both the tool and the docs. The tool presents these as a set of sequential steps:

image

**Comment by alexfedosov** [comment]
> if you're supposed to do the dump and restore and then follow that with LR to catch up

Exactly this

**Review by alexfedosov — APPROVED** [review]

**Review by danieltprice — APPROVED** [review]

**Pull request merged**